docs/getting-started/README.md

```diff
@@ -204,7 +204,7 @@ npx hardhat run scripts/sample-script.js --network localhost
 
 ---
 
-Congrats! You have created a project, ran a Hardhat task, compiled a smart contract, installed a Waffle integration plugin, wrote and ran a test using the Waffle and ethers.js plugins, and deployed a contract.
+Congrats! You have created a project, run a Hardhat task, compiled a smart contract, installed a Waffle integration plugin, written and run a test using the Waffle and ethers.js plugins, and deployed a contract.
```

Comment:
This is a question to learn from this: why is this in infinitive despite "created" being in simple past?

Reply:
In this case "run" is not an infinitive, but rather a past participle. Technically the form here is "have run."

Reply:
Ohhh! Makes sense! Thanks